### PR TITLE
Feature: Union schema compatibility

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
Issue: #12 

Confirm the following files were correctly updated automatically:
- [x] CHANGELOG
- [x] README
- [x] stg_tiktok_ads.yml, src_tiktok_ads.yml, tiktok_ads.yml (depending if source or transform)
- [x] docs.md
- [x] joins
- [ ] ~window functions (partition by)~
- [x] 'source_relation' column added in staging and transform models
- [x] tests

Manual updates:
- [x] Add `source_relation` to downstream models if necessary
- [x] Finish any incomplete/incorrect joins/partitions
- [x] Update tests to include `source_relation` in unique-combination-of-cols if necessary
    - May need to remove some uniqueness tests in favor of the combo test

Validation:
- [x] `dbt run` locally passes
- [x] `dbt test` locally passes

### CODE REVIEW

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:1]: Should we also include a callout regarding the renamed identifiers from the source? Would we be able to make a breaking change callout in this changelog to refer to the source release notes for details around breaking changes. Or is there a better approach you can think of to inform our users of the breaking changes in the upstream dependent package.

File: packages.yml
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]: Reminder to update before merge

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
These changes look great and run as expected when using production data locally!

However, I noticed the integration tests seem to be missing the seed data. Would you be able to make the appropriate corrections to ensure the integration tests are running properly against the seed data.

image

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:20]: Actually, can we remove the README "hide" sections in these updates. This way we are able to link directly to the section in the README where the users needs to understand how to configure the union feature.

Otherwise, I worry this cool feature is hidden from people actually using it.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Thanks for making the appropriate changes so our integration tests are running as expected!

This looks good to go with one final request to make a CHANGELOG and README update. Once those are applied then this will be good to merge and release once all others are ready.

File: CHANGELOG.md
Comment (fivetran-catfritz) [reply CHANGELOG.md]: Updated.

**Pull request merged**
